pkg/identityserver/emails/invitation.go

```diff
@@ -37,17 +37,13 @@ const invitationText = `Hello,
 
 You have been invited to join {{ .Network.Name }}.
 
-Your Invitation Token is: {{ .InvitationToken }}
+You can now go to {{ .Network.IdentityServerURL }}/register?invitation_token={{ .InvitationToken }} to register your user.
```

Comment:
```suggestion
You can now go to {{ .Network.IdentityServerURL }}/register?invitation_token={{ .InvitationToken }} to register your user.

After successful registration, you can access the Console here: {{ .ConsoleURL }}
```

Let's make it more clear by also referencing the console here (which is very likely what the user will be interested in after registration).